### PR TITLE
Query based sqlite connector

### PR DESCRIPTION
**Pull request by dmzmk**
- replaces 'append only' behavior on sqlite connector source side with user-provided query
- adds ability to truncate destination

### CODE REVIEW

**Review by alexandrinaw — APPROVED** [review]

File: myceliald/src/main.rs [hunk 1]
```diff
@@ -64,6 +63,7 @@ async fn run() -> Result<()> {
 
 #[tokio::main]
 async fn main() {
+    tracing_subscriber::fmt::init();
```
Comment (wwkeyboard) [inline myceliald/src/main.rs:66]: tracing?

**Review by wwkeyboard — APPROVED** [review]

File: myceliald/src/main.rs (hunk 1 above)
Comment (dmzmk) [reply myceliald/src/main.rs:66]: incoming
but if serious - forgot to drop it after checks

File: console/src/types.ts [hunk 2]
```diff
@@ -49,8 +49,8 @@ type SqlitePhysicalReplicationNodeData = BaseNodeData & {
 
 type SqliteConnectorNodeData = BaseNodeData & {
   path: string;
-  strict: boolean;
-  table: string;
+  origin: string;
```
[the remark this answers is not in the document]
Comment (dmzmk) [reply console/src/types.ts:52]: what would be better name?

File: myceliald/src/main.rs (hunk 1 above)
Comment (wwkeyboard) [reply myceliald/src/main.rs]: 👍

**Pull request merged**